make/photon/prepare/templates/docker_compose/docker-compose.yml.jinja

```diff
@@ -36,27 +32,15 @@ services:
     volumes:
       - {{data_volume}}/registry:/storage:z
       - ./common/config/registry/:/etc/registry/:z
-      - type: bind
```

Comment:
I'm not sure what is the benefit of `bind` over `volume` in this case :thinking: 
I've been using Harbor since I opened this PR with the changes I publish here without any issues. Just because I use volumes over binds will not necessarily create volumes on the system. If the source is a path that exists on the file system it will mount that inside the container at the target path (well even if it doesn't exist on the host it will do that by creating the folder in the source path). Difference is that if one uses bind over volumes is that volumes are taking a 3rd option, which in this case will be the `:z` flag for labeling the source for SELinux. Otherwise it would have to be done manually. Kinda the the whole gig of this PR really is to avoid doing that ehee ^-^